doc/getting_started/getting_started.rst

```diff
@@ -19,8 +19,8 @@ If you want to install the framework from source, use the following commands in
 
 .. code:: bash
 
-   wget https://raw.githubusercontent.com/ros-controls/ros2_control/{REPOS_FILE_BRANCH}/ros2_control.rolling.repos
-   vcs import src < ros2_control.repos
+   wget https://raw.githubusercontent.com/ros-controls/control.ros.org/master/ros_controls.{REPOS_FILE_BRANCH}.repos
+   vcs import src < ros_controls.{REPOS_FILE_BRANCH}.repos
```

Comment:
```suggestion
   wget https://raw.githubusercontent.com/ros-controls/control.ros.org/master/ros_controls.{DISTRO}.repos
   vcs import src < ros_controls.{DISTRO}.repos
```

I think it should be `{DISTRO}`, `{REPOS_FILE_BRANCH}` would be master instead of rolling